src/components/appointmentForm/AppointmentForm.js

```diff
@@ -0,0 +1,67 @@
+import React, { useState, useEffect } from 'react';
+
+import { ContactPicker } from '../contactPicker/ContactPicker';
+import DatePicker from "react-datepicker";
```

Comment:
`react-datepicker` wasn't included in the package.json. If you do plan on using it (which is okay as long as you explain it and link to docs), make sure to include it there

Reply:
John recommended `<input type"date">` which is obviously better. Gonna drop `react-datepicker`.